src/abstractdataframe/show.jl

```diff
@@ -249,7 +249,10 @@ function showrowindices(io::IO,
                 s = df[i, j]
                 strlen = ourstrwidth(s)
                 if ismissing(s)
-                    printstyled(io, s, color=:light_black)
+                    printstyled(io, "m", color=:light_black)
```

Comment:
This changes the printing of `missing`, right? I'd rather keep it as-is, since `m` isn't a real Julia object.

Reply:
sorry, that was just me playing around. can you reject that change or should I submit another PR?